Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -6911,23 +6911,15 @@ class BuildPlanTestCase: BuildSystemProviderTestCase {
     }
 }
 
-class BuildPlanNativeTests: BuildPlanTestCase {
-    override open var buildSystemProvider: BuildSystemProvider.Kind {
-        return .native
-    }
-
-    override func testDuplicateProductNamesWithNonDefaultLibsThrowError() async throws {
-        try await super.testDuplicateProductNamesWithNonDefaultLibsThrowError()
```

Comment:
Removed all overrides that just call back the corresponding super class method.

Reply:
Hmm, wasn't aware of that Linux hack, sorry :-(

So to be clear: all Linux test suites need to override *all* test cases... and then call `super`? If so, are we sure we actually did this hack everywhere? For instance `BuildPlanSwiftBuildTests` has dozen of tests but only two are overridden like this.

CC @cmcgee1024

Reply:
from my understanding, only 1 test needs to be overridden to have XCTest "populate" all the tests in the subclass.